tests/test_create_model.py

```diff
@@ -3,24 +3,27 @@
 import pytest
 
 from pydantic import BaseModel, ConfigDict, Extra, Field, ValidationError, create_model, errors
-from pydantic.decorators import field_validator
+from pydantic.decorators import field_validator, validator
 from pydantic.fields import ModelPrivateAttr
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_create_model():
-    model = create_model('FooModel', foo=(str, ...), bar=123)
+    model = create_model('FooModel', foo=(str, ...), bar=(int, 123))
```

Comment:
```
pydantic.errors.PydanticUserError: A non-annotated attribute was detected: `bar = 123`. All model fields require a type annotation; if 'bar' is not meant to be a field, you may be able to resolve this error by annotating it as a ClassVar or updating model_config["non_field_types"].
```
Same error you'd get for a non-annotated field outside of `create_model`. Do you want to add more detail if this happens in the context of `create_model`? If you want changes, any detail about how you'd like to handle it would be appreciated.

Either way, I can add a test for this in this file.

Reply:
that's fine I think.